<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Протокол предварительной записи пациента</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="CTextBrowser" name="txtClientInfoBrowser"/>
     <widget class="CBeforeRecordPopupMenu" name="tblActions"/>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CTextBrowser</class>
   <extends>QTextBrowser</extends>
   <header>library/TextBrowser.h</header>
  </customwidget>
  <customwidget>
   <class>CBeforeRecordPopupMenu</class>
   <extends>QTableView</extends>
   <header>Registry/BeforeRecordPopupMenu.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>478</x>
     <y>355</y>
    </hint>
    <hint type="destinationlabel">
     <x>501</x>
     <y>355</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>475</x>
     <y>344</y>
    </hint>
    <hint type="destinationlabel">
     <x>501</x>
     <y>345</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
